packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# 50.13.0 - January 13, 2021
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Added debug button to test sidebar to automatically run Apex Replay Debugger after test run ([PR #2804](https://github.com/forcedotcom/salesforcedx-vscode/pull/2804))
```

Comment:
Is the button named or an icon - and what does the hover text say? Just trying to determine if "Debug" button should be capitalized.

Reply:
It's an icon with text that should say "Debug Tests" or "Debug Single Test", depending what the tree item is.

Reply:
Ok, fine. Let's just leave it lowercase generic as you have it.